### ISSUE TITLE
Object Cache support

### ISSUE THREAD

**Issue by swissspidy** [pluginkollektiv/cachify#31]
Memcache (_not_ Memcache_d_) and Redis support would be nice via `advanced-cache.php`.

I think Simple Cache does this nicely.

**Comment by PixelbarEupen** [comment]
A bit late but i stumbled across this issue. The DB-Cache relies on transients that are cached. AFAIK transients will automatically be cached in memcache(d) or redis. 

I made a test on our server: without memcache, transients are generated and viewable in the DB. With memcache enabled, no transients found in the DB but the output is cached (visible in the footer notice).

**Comment by stklcode** [comment]
That's correct, the "DB" backend actually utilizes `set_transient()` which might end up in DB or an external object cache depending on the system setup.

Explicit _memcached_ (since 2.0.7, 2013) has a slightly different structure, so it can be included directly in the webserver config to skip all WP logic and the same holds for the recently added _Redis_ backend (which will be included in 2.4.0)

Both could be refactored to utilize WP's object cache abstraction and we should probably extend the documentation on the "DB" backend, but neither should have significant impact on the overall functionality.
